### PR TITLE
Laravel 6.1/6.2 support

### PR DESCRIPTION
**Pull request by emptynick**
This lets you install Voyager on top of Laravel 6.1 and 6.2.
No other steps are necessary.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -15,7 +15,7 @@
         }
     ],
     "require": {
-        "illuminate/support": "~5.5.0|~5.6.0|~5.7.0|~5.8.0|~6.0.0",
+        "illuminate/support": "~5.5.0|~5.6.0|~5.7.0|~5.8.0|~6.0.0|~6.1.0|~6.2.0",
```
Comment (fletch3555) [inline composer.json]: Can we just switch this to `"~5.5|~6.0"`?  Leaving off the 3rd number should allow it to match "5.*" and "6.*"

File: .travis.yml [hunk 2]
```diff
@@ -14,6 +14,7 @@ env:
     - LARAVEL_VERSION=5.7.*
     - LARAVEL_VERSION=5.8.*
     - LARAVEL_VERSION=6.0.*
+    - LARAVEL_VERSION=6.1.*
```
Comment (fletch3555) [inline .travis.yml]: Do we care about testing all the minor versions of 6? Now that they're following semver, we should be fine with just `6.*`

Comment (emptynick) [reply .travis.yml]: True, forgot that.

**Comment by emptynick** [comment]
All done

**Review by fletch3555 — APPROVED** [review]

**Pull request merged**